geonode/maps/api/views.py

```diff
@@ -21,7 +21,7 @@
 
 from dynamic_rest.viewsets import DynamicModelViewSet
 from dynamic_rest.filters import DynamicFilterBackend, DynamicSortingFilter
-
+from mapstore2_adapter.hooks import hookset
```

Comment:
We cannot assume the `geonode-mapstore-client` is installed here.

Let's use the Hookset interface in order to access the current hookset implementation.

Reply:
Hook added, please check again :)